src/farm/core/models/cv.py

```diff
@@ -0,0 +1,86 @@
+from attrs import define
+
+"""
+https://pdm-project.org/latest/
+
+ruff
+
+(Invoke-WebRequest -Uri https://pdm-project.org/install-pdm.py -UseBasicParsing).Content | python -
+
+
+"""
+
+@define
+class CameraObject:
+    width: float
+    height: float
+    scale: float
+    rotation: float
+
+@define
+class GenericAquarium(CameraObject):
+    center_coord_x: float
+    center_coord_y: float
+
+@define
+class AnalyzedAnimal:
+    x_pos: float
+    y_pos: float
+
+@define
+class Fish(AnalyzedAnimal):
+    shape = ""
+    width: float
+    height: float
+
+@define
+class ImageProcessor:
+    def fix_aquarium_rotation():
```

Comment:
add self for each method. If is intended to be static annotate with `@staticmethod`